frontend/src/metabase/parameters/utils/parameter-type.unit.spec.ts

```diff
@@ -31,4 +32,30 @@ describe("isSingleOrMultiSelectable", () => {
     });
     expect(isSingleOrMultiSelectable(parameter)).toBe(true);
   });
+
+  describe("parameters that have a template tags target (metabase#29997)", () => {
+    it("is false", () => {
+      const parameter = createMockUiParameter({
+        hasVariableTemplateTagTarget: true,
+      });
+      expect(isSingleOrMultiSelectable(parameter)).toBe(false);
+    });
+
+    it("is false even for parameters with acceptable types and corresponding subTypes", () => {
```

Comment:
I added 2 more scenarios from above that would return `true`, but `hasVariabletemplateTagTarget` should override that and return `false` instead.